src/modules/settings/AppSettings.js

```diff
@@ -71,6 +75,18 @@ class AppSettings {
   clearSelectedCity = async () => {
     await this.asyncStorage.removeItem(SELECTED_CITY_KEY)
   }
+
+  setIntroShown = async () => {
+    await this.asyncStorage.setItem(INTRO_SHOWN_KEY, JSON.stringify(true))
+  }
+
+  loadIntroShown = async (): Promise<boolean> => {
+    const value = await this.asyncStorage.getItem(INTRO_SHOWN_KEY)
+    if (value === null && process.env.E2E_TEST_IDS) {
```

Comment:
Checking for this environment var leads to failing unit tests, because it's also set for unit tests by jenkins. @maxammann any workaround? Maybe we should move unit tests/linting to a different stage?

Reply:
This seems like a bug in the jenkinsfile. You can try to separate the stages.

Note: the .env. variables geht replaced by babel.